shared/language-specs/languages.ts

```diff
@@ -28,6 +28,14 @@ const clojureSpec: LanguageSpec = {
     commentStyles: [lispStyleComment],
 }
 
+const cobolSpec: LanguageSpec = {
+    languageID: 'cobol',
+    stylized: 'Cobol',
+    fileExts: ['cbl', 'cob', 'cpy', 'dds', 'ss', 'wks', 'pco'],
+    identCharPattern: createIdentCharPattern('-'),
```

Comment:
question:
in general how does this handle which chars can be first in an identifier and which can only appear after the first char?

Reply:
Right now this is only used to scan left/right from the current hover position eating characters while it can on each side. We don't currently support a way to make special first characters (java idents can't start with a number, for example, but we'd match it since numbers are in the regex).

Reply:
so basically this would consider an integer literal an identifier, but that doesn't actually affect usability?

Reply:
Yes, on defs it won't match any output of ctags, and without a def you can't find references either.

If you'd like you can open an issue to improve identifier selection. There's probably a lot of little heuristic tweaks we can do to improve this.